modules/Bio/EnsEMBL/Compara/PipeConfig/DumpHomologiesByMLSS_conf.pm

```diff
@@ -26,14 +26,15 @@ Bio::EnsEMBL::Compara::PipeConfig::DumpHomologiesByMLSS_conf
 
 =head1 SYNOPSIS
 
-    init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::DumpPerMLSSHomologies_conf -member_type ncrna -clusterset_id murinae
+    init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::DumpPerMLSSHomologies_conf -host mysql-ens-compara-prod-X -port XXXX \
+        -division $COMPARA_DIV -member_type ncrna -clusterset_id murinae
 
     By default the pipeline dumps the database named "compara_curr" in the
     registry, but a different database can be selected with -rel_db
 
 =head1 DESCRIPTION
 
-    This pipeline dumps all the gene-trees and homologies under #base_dir#
+This pipeline dumps all the gene-trees and homologies under #base_dir#.
```

Comment:
Not worth fixing, but in some cases you remove leading whitespace and in some, you add it. Not sure what the rules are ;)

Reply:
It's indeed confusing. From the other reviews I had the feeling you wanted to remove the leading whitespace but in this PR you've kept it in several files

Reply:
Sorry! Will try to find all that I have left and remove them to set some kind of consistency among myself at least.